_data/wgs.yml

```diff
@@ -2,68 +2,70 @@
   bottomliner: Farbod Saraf
   slug: leadership
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "What does it take to be an open source leader? How do leaders emerge, and what can we learn about what makes leaders effective? What are the pressures and issues that leaders face, and how can we mitigate them through active development?"
 
 - title: Open Source Funders
   bottomliner: Richard Littauer
   slug: open-source-funders
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "There's no current database of how much money goes into open source: we want to build this. Follow the money - where does it come from, where does it go, and how useful are different avenues of funding for open source projects at scale."
 
 - title: Sustainability Readiness
   bottomliner: Erin McKean
   slug: sustainability-readiness
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "How can we ensure that a new open source project is born sustainable? What steps can be put into place to fight entropy in both code and governance structures in open source?"
 
 - title: Licensing
   bottomliner: Benjamin Nickolls
   slug: licensing
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "What is licensing, anyway? What's the current state of the ecosystem, where are we going, and how can we talk about licenses without flipping over the proverbial dinner table?"
 
 - title: Personas on Sustainability
   bottomliner: Jacob Redding
   slug: personas
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "Who are sustainers, anyway? What does it mean to sustain a project? How can we use persona development to learn more about the needs of all of the stakeholders in open source?"
 
 - title: 2020 Sustain Summit Report
   bottomliner: Benjamin Nickolls
   slug: 2020-summit-report
   status: Incubating
-  ranking: 1
+  ranking: 0
```

Comment:
This hasn't had it's first meeting yet, and no work has been done. Keep it in incubating.